.github/workflows/nightly-build.yml

```diff
@@ -52,10 +73,9 @@ jobs:
       - name: Download natives and build project
         uses: actions/download-artifact@v4
         with:
-          pattern: natives-*
           path: ./cli/target/
       - name: Deploy to OSSRH nexus
         env:
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-        run: mvn --settings .mvn/settings.xml -DskipTests=true -Darchetype.test.skip=true -Dgpg.skip=true -Dstyle.color=always -B -ntp -Passembly,deploy deploy
+        run: mvn --settings .mvn/settings.xml -DskipTests=true -Darchetype.test.skip=true -Dgpg.skip=true -Dstyle.color=always -B -ntp -Passembly,msi,deploy deploy
```

Comment:
It looks like the MSI is attached into the deployment. I haven't change the `release` workflow yet. Maybe we can first run the Nightly Build and then adapt the changes to the release workflow

https://github.com/alfeilex/IDEasy/actions/runs/13546792586/job/37860853837#step:5:1240